checkout.js

```diff
@@ -1,5 +1,9 @@
+function onSaleItems(item) {
+  return item.discount > 0
+}
+
 function selectSaleItems(items) {
-  return items
+  return items.filter(onSaleItems)
```

Comment:
You could also inline the function as well. Either works.

```
return items.filter(function (item) {
   return item.discount > 0
})
```

And if you inline it you can also make use of the ES6 syntax to get the implicit return.

```
return items.filter((item) => item.discount > 0 )
```